database.sql

```diff
@@ -55,13 +55,13 @@ CREATE TABLE IF NOT EXISTS `auth_codes` (
 -- TABLE cache
 --
 CREATE TABLE IF NOT EXISTS `cache` (
-	`k` varbinary(255) NOT NULL COMMENT '',
-	`v` mediumtext COMMENT '',
-	`expire_mode` tinyint unsigned NOT NULL DEFAULT 0 COMMENT '',
-	`updated` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT '',
-	 PRIMARY KEY(`k`),
-	 INDEX `expire_mode_updated` (`expire_mode`,`updated`)
-) DEFAULT COLLATE utf8mb4_general_ci;
+	`k` varbinary(255) NOT NULL COMMENT 'cache key',
+	`v` mediumtext COMMENT 'cached serialized value',
+	`expires` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT 'datetime of cache expiration',
+	`updated` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT 'datetime of cache insertion',
+	PRIMARY KEY (`k`),
+	KEY `k_expires` (`k`,`expires`) USING BTREE
```

Comment:
Just some side note: Have you created this file by hand?

Reply:
No, it's phpMyAdmin output, why?

Reply:
Please create it by calling ```scripts/dbstructure.php > database.sql```. This is unified - and easier to perform.

Reply:
Ah, I forgot about that, will do tonight.